VS2017/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM microsoft/windowsservercore:10.0.14393.1944
+
+#ENV MSVS_INSTALL_URL=https://aka.ms/vs/15/release/vs_community.exe \
+#    MSVS_INSTALL_EXECUTABLE=vs_community.exe
+
+#RUN powershell "(New-Object System.Net.WebClient).DownloadFile(\"$env:MSVS_INSTALL_URL\", \"$env:TMP\$env:MSVS_INSTALL_EXECUTABLE\")" && \
```

Comment:
Why are these lines commented out? Could there be a comment in the file to explain all the code that is commented out?